sns_ik_examples/src/ik_tests.cpp

```diff
@@ -323,7 +342,12 @@ void test(ros::NodeHandle& nh, double num_samples_pos, double num_samples_vel,
       double elapsed = 0;
 
       start_time = boost::posix_time::microsec_clock::local_time();
-      rc=snsik_solver.CartToJnt(JointSeed[i],end_effector_pose,result);
+
+      if(nullspace_bias_task)
+        rc = snsik_solver.CartToJnt(JointSeed[i], end_effector_pose, ns_bias, result);
```

Comment:
Between optional parameters, nope. But `result` is always required, so it should not be last.